src/Config/CS_SDK.props

```diff
@@ -23,16 +23,27 @@
     <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
     <DebugSymbols>true</DebugSymbols>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)' == 'Debug'">
-    <DebugType>full</DebugType>
+  <PropertyGroup Condition="'$(Configuration)' == 'Debug' AND $(Platform.Contains('NET60')) ">
+    <DebugType>portable</DebugType>
```

Comment:
I think you could just add
```
<DebugType Condition="$(Platform.Contains('NET60')) ">portable</DebugType>
<DebugType Condition="!$(Platform.Contains('NET60')) ">full</DebugType>
```
If you think this is more readable

Reply:
yes, it's nicer, also considering that Debug or Release changes nothing, maybe I can move these elements outside the debug and release prop groups and not repeat them.